lib/sufia/import/collection_translator.rb

```diff
@@ -0,0 +1,35 @@
+module Sufia::Import
+  # Imports a Sufia 6.0-exported Collection into a Sufia PCDM Collection
+  class CollectionTranslator
+    def initialize(settings)
+      @collection_builder = CollectionBuilder.new(settings)
+    end
+
+    # loop through the files found in import_dir and map each Collection
+    # @param import_dir a directory containing export files, e.g.: collection_[id].json
+    # @return list of imported files
+    def import(import_dir, filename_prefix)
```

Comment:
@hackmastera This code for opening files seems quite simmilar to the GenericFileTranslator: https://github.com/projecthydra/sufia/blob/7.2-migration/lib/sufia/import/generic_file_translator.rb#L17-L23

May indicate that a base class would be nice.